README.md

```diff
@@ -29,14 +30,19 @@ Core Server (i.e. SERVER) project are **public**.
 ## Installation
 This library works with **Swift 5.2+** , and supports Linux and macOS usage. The minimum macOS version required is **10.15**.
 
-Installation is supported via [Swift Package Manager](https://swift.org/package-manager/).
+### Using in a New Project
+To create a new project using the library, the easiest way to get started is by using
+Vapor's command line tool, [Vapor Toolbox](https://github.com/vapor/toolbox), along with our application template:
+```
+vapor new MyProject --template https://github.com/mongodb/mongodb-vapor-template/
+```
+This will create a new project from a template, which you can edit to your liking. See the instructions [here](https://github.com/mongodb/mongodb-vapor-template/blob/main/README.md) or in the generated README for more details on the generated project.
 
-### Step 1: Install Required System Libraries (Linux Only)
+Note that for use on Linux you will need to install some additional dependencies (see [below](#step-1--install-required-system-libraries--linux-only-).) 
+### Using in an Existing Project
+#### Step 1: Install Required System Libraries (Linux Only)
 The driver vendors and wraps the MongoDB C driver (`libmongoc`), which depends on a number of external C libraries when built in Linux environments. As a result, these libraries must be installed on your system in order to build MongoSwift.
-
-To install those libraries, please follow the [instructions](http://mongoc.org/libmongoc/current/installing.html#prerequisites-for-libmongoc) from `libmongoc`'s documentation.
```

Comment:
This line still needs to be here I think, otherwise it might not be clear which libraries need to be installed.